### ISSUE TITLE
Standarize classnames

### ISSUE THREAD

**Issue by kavinsky** [michail-nikolaev/task-force-arma-3-radio#1131]
All our current prefix classnames are nor standard, some are `TFAR_` and others `tf_`

Why should we do this? standarize all our configs and codebase.

Why we shouldnt do this? break legacy scripts/loadouts/missions of all our users.

Comments?

**Comment by dedmen** [comment]
Keep scripts working by adding classes of the old name that will get
replaced with the new one ?
tf_test : tfar_test{
Scope = 2;
}

**Comment by CorruptedHeart** [comment]
We should work to standardise with `TFAR_` but should attempt to maintain backwards compatibility somewhat.

**Comment by kavinsky** [comment]
yeah what dedmen suggest its good but should notice the player about deprecation dates.

**Comment by dedmen** [comment]
I have all classname changes done. And backwards-compat is good.
Questions:
Name radios TFAR_radio_XX instead of TFAR_XX ? IMO not needed
Also rename all Variables/? Variables will probably break a ton of user scripts so thats a bad Idea.
Also i renamed all #defines from TF_ to TFAR_
a problem i found is that we currently cant distinguish defines from variables. I would like to use lowercase variables and uppercase defines. But most variables in ClientInit start with uppercase and thats a bother. If you want me to do that tell me.
But if we keep variables TF_ anyway and defines TFAR_ that would make them easy to differentiate

**Comment by dedmen** [comment]
#TODO remove TFAR_ from macros. So they are easy to differentiate
